plutus-pab/src/Cardano/Node/Types.hs

```diff
@@ -70,17 +199,16 @@ initialAppState wallets =
         , _followerState = initialFollowerState
         }
 
-newtype NodeFollowerState = NodeFollowerState { _unNodeFollowerState :: Map FollowerID Int }
-    deriving (Show)
-
-_NodeFollowerState :: Iso' NodeFollowerState (Map FollowerID Int)
-_NodeFollowerState = iso _unNodeFollowerState NodeFollowerState
-
+-- | Empty initial 'NodeFollowerState'
 initialFollowerState :: NodeFollowerState
 initialFollowerState = NodeFollowerState Map.empty
 
-newtype FollowerID = FollowerID Int
-    deriving stock (Show, Eq, Ord, Generic)
-    deriving newtype (ToJSON, FromJSON, ToHttpApiData, FromHttpApiData, Integral, Enum, Real, Num, Pretty)
+-- | 'ChainState' with initial values
+initialChainState :: Trace.InitialDistribution -> ChainState
+initialChainState =
+    view EM.chainState .
+    MultiAgent.emulatorStateInitialDist . Map.mapKeys EM.walletPubKey
 
-makeLenses 'AppState
+-- | Isomorphism to go between 'NodeFollowerState' and 'Map FollowerID Int'
+_NodeFollowerState :: Iso' NodeFollowerState (Map FollowerID Int)
```

Comment:
Does this adhere to a naming convention of some sort? `_NodeFollowreState` seems odd to me for this iso but i am maybe just not aware..

Reply:
Yes the name is correct but you could use [`makePrisms`](http://hackage.haskell.org/package/lens-4.15.4/docs/Control-Lens-TH.html#v:makePrisms) which generates isos when possible.